app/bundles/CoreBundle/Helper/DateRelativeParser.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\CoreBundle\Helper;
+
+class DateRelativeParser
+{
+    /** @var array */
+    private $dictionary;
+
+    /** @var string */
+    private $timeframe;
+
+    /** @var string */
+    private $prefix;
+
+    /**
+     * DateRelativeParser constructor.
+     *
+     * @param array  $dictionary
+     * @param string $timeframe
+     */
+    public function __construct(array $dictionary, $timeframe, $prefixes = [])
```

Comment:
```suggestion
    public function __construct(array $dictionary, $timeframe, array $prefixes = [])
```
Also $prefixes is missing in the method comment.